guides/common/modules/configuring_satellite_external_dns.adoc

```diff
@@ -17,6 +17,7 @@ The following example configures a domain `virtual.lan` as one subnet 192.168.38
 +
 [options="nowrap"]
 ----
+// Check for Debian
```

Comment:
Debian stores config in `/etc/bind` (compared to `/etc` for RedHat). vardir is `/var/cache/bind` instead of `/var/named`. https://github.com/theforeman/puppet-dns/blob/master/manifests/params.pp might be useful as a reference.